lib/server.js

```diff
@@ -136,15 +145,29 @@ module.exports = function createServer(config, log) {
 
             var retryAfter = [blockEmail, blockIpEmail, blockIp].reduce(max)
 
+            var allowIP = ip in allowedIPs.ips
+            var allowedEmailDomain = allowedEmailDomains.isAllowed(email)
```

Comment:
naming nit: `allowedIP` rather than `allowIP` for consistency with the other names in this block.